_posts/2015-06-09-mac-os-x-ruby-ruby-install-chruby-and-you.markdown

```diff
@@ -36,22 +36,12 @@ First of all, we're going to need to install some package management script so t
 
     ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-Next, we'll need to install `wget` because some of the commands we're about to run use it.
+Next, we'll install `chruby`, `ruby-install` and `wget`.
 
-    brew install wget
+    brew install chruby ruby-install wget
```

Comment:
Do we need to install `wget` here if we're installing everything else from brew? I think `wget` is only a dep because we're fetching `ruby-install` and `chruby` using it.